lib/puppet/face/certificate.rb

```diff
@@ -3,9 +3,9 @@
 
 Puppet::Indirector::Face.define(:certificate, '0.0.1') do
   copyright "Puppet Labs", 2011
-  license   "Apache 2 license; see COPYING"
+  license   _("Apache 2 license; see COPYING")
 
-  summary "Provide access to the CA for certificate management."
+  summary _("Provide access to the CA for certificate management.")
   description <<-EOT
```

Comment:
I was told to leave docs-like strings alone. So I should probably go back and unwrap the previous ones...